Build/runpssa.ps1

```diff
@@ -0,0 +1,71 @@
+[cmdletbinding()]
+param (
+    [Parameter(Mandatory)]
+    [string]$PSD1Path,
+
+    [Parameter(Mandatory)]
+    [string]$SettingsFile,
+
+    [Parameter()]
+    [string]$OutputXML = "$($env:AGENT_WORKFOLDER)\PssaResults.xml"
+)
+
+begin {
+    try {
+        $null = Get-PackageProvider -Name Nuget
+    }
+    catch {
+        $null = Install-PackageProvider -Name Nuget -Scope CurrentUser -Force -Confirm:$false
+    }
+
+    if (@(Get-Module -ListAvailable Pester)[0].Version -lt [version]'4.4.0') {
+        $null = Install-Module -Name Pester -Scope CurrentUser -Force -Confirm:$false -SkipPublisherCheck
+    }
+
+    if (@(Get-Module -ListAvailable PSScriptAnalyzer)[0].Version -lt [version]'1.17.1') {
+        $null = Install-Module -Name PSScriptAnalyzer -Scope CurrentUser -Force -Confirm:$false
+    }
+
+    Import-Module Pester -Version 4.4.0
+    Import-Module PSScriptAnalyzer -Version 1.17.1
+}
+
+process {
+    $moduleBase = Split-Path $PSD1Path -Parent
+    $moduleName = (Split-Path $PSD1Path -Leaf) -replace '.psd1$'
+
+
+    $testFile = New-TemporaryFile
+    Rename-Item -Path $testFile -NewName "$($testFile.BaseName).ps1"
+    $testFilePS1 = "$($testFile.Directory)\$($testFile.basename).ps1"
+
+    $testFileContent = @"
```

Comment:
Use a literal here-string `@'` instead; will save us a _lot_ of pesky backticks here. 😄